extensions/default/src/ViewerLayout/Header.jsx

```diff
@@ -3,10 +3,28 @@ import PropTypes from 'prop-types';
 // TODO: This may fail if package is split from PWA build
 import { useHistory } from 'react-router-dom';
 //
-import { NavBar, Svg, Icon, IconButton } from '@ohif/ui';
+import { NavBar, Svg, Icon, IconButton, Dropdown, useModal } from '@ohif/ui';
 
 function Header({ children }) {
   const history = useHistory();
+  const { show } = useModal();
+
+  const showAboutModal = () => {
+    const modalComponent = () => <div>About modal</div>;
+    show({
+      title: 'About',
```

Comment:
I'm not sure but translations will be added in separate tasks?

Reply:
We're not adding it yet, but I added it anyway to avoid untranslated texts throughout the application 👍